docs/docs/using-gatsby-image.md

```diff
@@ -107,9 +107,10 @@ This GraphQL query creates multiple sizes of the image and when the page is rend
 
 So this is all very nice and it’s far better to be able to use this from NPM vs. implementing it yourself or cobbling together several standalone libraries.
```

Comment:
```suggestion
So this is all very nice and it’s far better to be able to use this from npm vs. implementing it yourself or cobbling together several standalone libraries.
```

This is a little nitpicky but I think both npm and webpack have their name lowercase according to their brand guidelines